type:  "babbling"
sequential:  true
data_structure:
  depth:  "depth.yml"
  rgb:  "rgb.yml"
  motion:  "motion_rects.yml"
  joints_values:  "controller_feedback.yml"
  dataset_file:  "dataset.yml"
  gmm_archive_file: "gmm_archive"
topics:
  data:
    motion: "/object_babbling/motion_detector_node"
    joints_values: "/object_babbling/controller_node/controller_node/feedback"
    dataset: "/object_babbling/babbling_node/dataset"
    classifier: "/object_babbling/babbling_node/classifier"
  servers:
    controller_server: "controller_node"
    database_server: "/object_babbling/db_manager_node"
  services:
    motion_detector_service: "/object_babbling/motion_detector_node/is_moving"
